ui/app/rif/pages/configuration/index.js

```diff
@@ -131,51 +141,64 @@ class RifConfiguration extends Component {
 
   render () {
     if (this.state.loading) {
-      return (<div>Loaing Configuration...</div>);
+      return (<div className="">Loaing Configuration...</div>);
```

Comment:
Are we putting some classNames here? Or just leave it empty? Delete it in that case.

Reply:
Done